specification/automation/resource-manager/Microsoft.Automation/preview/2022-06-30-preview/runtime.json

```diff
@@ -0,0 +1,422 @@
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/runtimes": {
+      "get": {
+        "tags": [
+          "Runtime"
+        ],
+        "operationId": "Runtime_ListByAutomationAccount",
+        "description": "Retrieve a list of runtimes.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/runtimeconfigurationoperations"
+        },
+        "x-ms-examples": {
+          "List runtime by Automation Account": {
+            "$ref": "./examples/listRuntimes.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RuntimeListResult"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/Runtime"
+      }
+    }
+  },
+  "definitions": {
+    "Runtime": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets or sets the id of the resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets or sets the name of the runtime."
+        },
+        "properties": {
+          "$ref": "#/definitions/RuntimeProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the hybrid worker group properties."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Resource system metadata."
+        }
```

Comment:
Use Proxy resource from common types for this.

Reply:
Done